shells/plain/container.js

```diff
@@ -36,10 +36,13 @@ if (iframeSrc) {
 
 window.addEventListener('keydown', function(e) {
   if (e.altKey && e.keyCode === 68) { // Alt + D
-    if (document.body.className === 'devtools-bottom') {
-      document.body.className = 'devtools-right';
-    } else {
-      document.body.className = 'devtools-bottom';
+    var documentBody = document.body;
```

Comment:
`document.body` should be safe and reliable in practice. Rather than adding indirection to the code, an early throw if it is not present is a cleaner way to satisfy flow.